src/compiler/parser.ts

```diff
@@ -4720,10 +4720,16 @@ module ts {
                         amdModuleName = amdModuleNameMatchResult[2];
                     }
 
-                    var amdDependencyRegEx = /^\/\/\/\s*<amd-dependency\s+path\s*=\s*('|")(.+?)\1/gim;
+                    var amdDependencyRegEx = /^\/\/\/\s*<amd-dependency/gim;

```

Comment:
there should be a space after "amd-dependency" and before path or name. i would add \s+ back to the regex